rotkehlchen/api/v1/schemas.py

```diff
@@ -2567,14 +2571,79 @@ def transform_data(
             raise ValidationError(f'Could not deserialize data: {e!s}') from e
 
 
-class LocationAssetMappingsUpdateSchema(LocationAssetMappingsBaseSchema):
+class LocationAssetMappingsUpdateSchema(Schema):
     entries = NonEmptyList(fields.Nested(LocationAssetMappingUpdateEntrySchema), required=True)
 
 
-class LocationAssetMappingsDeleteSchema(LocationAssetMappingsBaseSchema):
+class LocationAssetMappingsDeleteSchema(Schema):
     entries = NonEmptyList(fields.Nested(LocationAssetMappingDeleteEntrySchema), required=True)
 
 
+class CounterpartyAssetMappingsBaseSchema(Schema):
+    counterparty = fields.String(
+        required=True,
+        validate=webargs.validate.OneOf(choices=[CPT_HYPER]),
```

Comment:
That's not what I meant. Valid values should be any valid counterparty value. We do have `self.rotkehlchen.chains_aggregator.get_all_counterparties()` which should return it.